pandas/io/tests/test_gbq.py

```diff
@@ -557,6 +557,21 @@ def test_zero_rows(self):
         expected_result = DataFrame(page_array, columns=['title', 'id'])
         self.assert_frame_equal(df, expected_result)
 
+    def test_standard_sql(self):
+        standard_sql = "SELECT DISTINCT id FROM " \
```

Comment:
can you add a test where you validate ``legacy``? (e.g. that it must be standard/legacy in lowercase)